relay-general/src/protocol/event.rs

```diff
@@ -243,6 +243,8 @@ pub struct Event {
     pub fingerprint: Annotated<Fingerprint>,
 
     /// Custom culprit of the event.
+    ///
+    /// This field is deprecated and shall not be set by client SDKs.
     #[metastructure(max_chars = "culprit", pii = "maybe")]
```

Comment:
```suggestion
    #[metastructure(max_chars = "culprit", pii = "maybe", omit_from_schema)]
```

Reply:
will need more snapshot updates...

Reply:
wait, i actually have questions about this.  omitting it from the schema means that it will no longer show up on https://develop.sentry.dev/sdk/event-payloads/types/ right?  But the field is present in e.g. the rust SKD: https://docs.rs/sentry/0.23.0/sentry/protocol/struct.Event.html and this means omitting it from the schema makes it again impossible to discover what that field is for.

So should it really be omitted from the schema?

Reply:
force-pushed to reset to before this commit